service/cart_http_test.go

```diff
@@ -0,0 +1,218 @@
+package service
+
+import(
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+	"joshsoftware/go-e-commerce/db"
+	"net/http"
+	"errors"
+)
+
+type Cart struct {
```

Comment:
why there is a need of defining a new struct here?

Reply:
same as above. Now removed